src/Settings.ts

```diff
@@ -78,12 +78,15 @@ module Settings
     }
 
     export function getUrlVars()
-    {
+	{
+		//TODO
+		//VERIFICAR O RETORNO QUE PODE ESTAR INCORRETO PARA O METODO
         var vars = {};
         window.location.href.replace(/[?&]+([^=&]+)=([^&]*)/gi, function (m, key, value) 
         {
-            vars[key] = value;
-            return true;
+			vars[key] = value;
+			//return true;
+            return "";
```

Comment:
Whats this change about?

Reply:
Visual studio says the method could not return true just a string, then i setted it to return "", and check later.
I checked it in the chrome console now, and it works perfectly to get the vars after ? or & in url